homeassistant/components/alexa/capabilities.py

```diff
@@ -1110,6 +1114,30 @@ def capability_resources(self):
             )
             return self._resource.serialize_capability_resources()
 
+        # Input Number Value
+        if self.instance == f"{input_number.DOMAIN}.{input_number.ATTR_VALUE}":
+            min_value = float(self.entity.attributes.get(input_number.ATTR_MIN))
```

Comment:
If we know it's always there, use square brackets. `get` could return `None`, in which case it's more difficult to trace the key was missing, Square brackets raise `KeyError` if it's not there. We should always do this if we know the attribute exists.

```suggestion
            min_value = float(self.entity.attributes[input_number.ATTR_MIN])
```

However, in this case, isn't this already a number ?

Reply:
Thanks for the recommendation on this.

We expect the attribute to return a number, but I cast any input to/from the Alexa API as a `float` for this instance because of the user configured `precision` attribute. As opposed to `int` for other `RangeController` instances that deal primarily with percentages units.

I previously ran into a bug with the `AdjustRangeValue` directive handler casting the incoming `rangeValueDelta` payload as a `string` and trying to calculate the result against the current `int` value. Therefore I made sure to cast each input/output value for this controller so nothing was left to chance.